src/help/zaphelp/contents/start/concepts/authentication.html

```diff
@@ -36,6 +36,17 @@
 		Response View and select the text you wish to define as the indicator using the
 		mouse and select the <i>Flag as Context... Logged in/out indicator</i> right-click menu option.
 	</p>
+	
+	<p> The logged in indicator is not mandatory (none of the indicators are),
```

Comment:
I just thought it added clarity, but I see your point. I'll tweak/merge the two.